front50-api/front50-api.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
+dependencies {
+  implementation project(":front50-core")
+  implementation "org.springframework:spring-context"
```

Comment:
I think this is being used for the `Errors` class in the validator interface. I would work around this class somehow (you'll have to refactor the other `Validator`s). I'm happy to brainstorm strategies with you.

The reasoning behind limiting dependencies is that a plugin developer should be able to depend on the interfaces in the `front50-api` package until there's a breaking change in the interface. If this package brings its own version of Spring, then the plugin will stop working whenever that version conflicts with the version of Spring inside Front50.